generators/entity-client/index.js

```diff
@@ -95,6 +97,7 @@ module.exports = class extends EntityClientGenerator {
                             mtUtils.processPartialTemplates(angularFiles.entityTenantAwareTemplates(this), this);
                             break;
                         case 'react':
+                            mtUtils.processPartialTemplates(reactFiles.templates(this), this);
```

Comment:
This shouldn't be called here as this for a tenant Aware entity not the tenant itself.